user/service/client/client.go

```diff
@@ -95,9 +95,9 @@ func (c *Client) Delete(ctx context.Context, id string, deleet *user.Delete, con
 		return false, err
 	}
 
-	session := c.UserStructuredStore().NewUserRepository()
+	repository := c.UserStructuredStore().NewUserRepository()
 
-	result, err := session.Get(ctx, id, condition)
+	result, err := repository.Get(ctx, id, condition)
```

Comment:
We should use `context.Background()`, otherwise the context will be terminated for requests which time out. This will be deprecated soon, but I think it's still important.

Reply:
Ah yes, good point!